src/proto/mgmt/pool.proto

```diff
@@ -33,6 +33,7 @@ message PoolCreateReq {
 	repeated uint32 ranks = 12; // target ranks (manual config)
 	uint64 scmbytes = 13; // SCM size in bytes (manual config)
 	uint64 nvmebytes = 14; // NVMe size in bytes (manual config)
+	uint32 policy = 15; // tier placement policy (manual config)
```

Comment:
Might be better to name this tier_policy so that we don't need to rename it later if we add support for other kinds of policies. We're trying to standardize the protobuf field names on snake_case as it results in more consistent JSON serialization.